tests/jeos/image_info.pm

```diff
@@ -50,79 +72,136 @@ sub run {
+    my $size_mb = $image_size / 1024 / 1024;
     record_info('Image', "Image: $image\nSize: $size_mb");
 
-    # Record this value in the InfluxDB
+    # Get list of packages installed in the system
+    my $packages = script_output('rpm -qa --queryformat "%{SIZE} %{NAME}\n" |sort -n -r');
+    my @rpm_array = split(/\n/, $packages);
+    my $num_packages = scalar @rpm_array;
+    record_info('rpm total', "Total number of installed packages: $num_packages");
+    record_info('rpm list', $packages);
+
+    # Get the size of different file types
+    my $btrfs_summary = script_output('btrfs filesystem df --mbytes --si /');
+    record_info('btrfs', "$btrfs_summary");
+    my ($data) = $btrfs_summary =~ /Data.*/g;
+    my ($data_mb) = $data =~ /(\d+\.\d+)/g;
+    my ($system) = $btrfs_summary =~ /System.*/g;
+    my ($system_mb) = $system =~ /(\d+\.\d+)/g;
+    my ($metadata) = $btrfs_summary =~ /Metadata.*/g;
+    my ($metadata_mb) = $metadata =~ /(\d+\.\d+)/g;
+    my ($globalreserve) = $btrfs_summary =~ /GlobalReserve.*/g;
+    my ($globalreserve_mb) = $globalreserve =~ /(\d+\.\d+)/g;
+    record_info('Data', "$data_mb MB");
+    record_info('System', "$system_mb MB");
+    record_info('Metadata', "$metadata_mb MB");
+    record_info('GlobalReserve', "$globalreserve_mb MB");
+
+    # Get size of the root directories in KB (except /proc).
+    my $dirs = script_output('du -d 1 --block-size=1K --exclude=/proc /');
+    record_info("dirs", $dirs);
+
+    # Record all this information in the InfluxDB
     # We only allow this on certain conditions:
+    #  - The job must contain INFLUXDB_SERVER, _SECRET_INFLUXDB_USER and _SECRET_INFLUXDB_PWD
+    #  - The job shouldn't be a verification run
     #  - The job must be executed from OSD or O3
-    #  - The job must contain INFLUXDB_SERVER and _SECRET_INFLUXDB_USER and _SECRET_INFLUXDB_PWD
-    #  - The job shouldn't be a verification run => CASEDIR variable
     my $influxdb_server = get_var('INFLUXDB_SERVER');
     my $influxdb_user = get_var('_SECRET_INFLUXDB_USER');
     my $influxdb_pwd = get_var('_SECRET_INFLUXDB_PWD');
+    return unless ($influxdb_server);
+    unless ($influxdb_user || $influxdb_pwd) {
+        record_info('INFO', 'INFLUXDB_SERVER variable is set but "_SECRET_INFLUXDB_USER" and "_SECRET_INFLUXDB_PWD" are missing.');
+        return;
+    }
+    # To not let openQA show the echo command that exposes user/password on the UI, we need to make
+    # a local perl call and store it to a file in the worker and upload it to the VM
+    system("echo '-u $influxdb_user:$influxdb_pwd' > $dir/os-autoinst-distri-opensuse/data/influxdb_conf");
+    assert_script_run('curl -s ' . data_url('influxdb_conf') . ' -o /root/influxdb_conf');
 
-    return if (get_var('CASEDIR') !~ m/^sle$|^opensuse$/);
-    return unless ($influxdb_server || $influxdb_user || $influxdb_pwd);
+    # CASEDIR var is always set, and if when no specified, the value is "sle" for OSD and "opensuse" for O3
```

Comment:
`It's always set, but if not ...` doesn't make much sense. Also: `if when no specified`

Reply:
Even if you don't inject manually `CASEDIR`, but default all jobs have that variable by default.
See https://openqa.opensuse.org/tests/2337465/file/vars.json

Reply:
In that case you can just use `get_required_var` like in other places and remove the comment